lib/heapwatch.js

```diff
@@ -107,6 +110,7 @@ class HeapWatch {
     }
 
     close() {
+        this.gcStats.removeListener('stats', this.reportStatsHandler);
```

Comment:
The 'require' of the gc-stats package was surrounded with try-catch because it's an optional binary dependency, thus requiring it might error out, thus the gcStats will be null, thus this will fail during shutdown.

Reply:
Makes sense. I'll add a check for it.